app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/MappingWizardDatastoresStep.js

```diff
@@ -33,6 +33,40 @@ class MappingWizardDatastoresStep extends React.Component {
 
   componentDidMount() {}
 
+  componentWillReceiveProps(nextProps) {
+    const {
+      showAlertAction,
+      isRejectedSourceDatastores,
+      isRejectedTargetDatastores
+    } = this.props;
+
+    const { selectedCluster, selectedClusterMapping } = this.state;
+    if (
+      isRejectedSourceDatastores !== nextProps.isRejectedSourceDatastores &&
+      nextProps.isRejectedSourceDatastores
+    ) {
+      showAlertAction(
+        __(
+          `Error retrieving cluster datastores: ${selectedCluster.name}, ID: ${
+            selectedCluster.id
+          }`
```

Comment:
I think we're supposed to be using `sprintf` to translate dynamic strings

Reply:
interesting... not really sure what the distinction is there yet...

Reply:
will update...